### PR TITLE
Cover yaml examples

### PR DESCRIPTION
**Pull request by lisyukanna**

### CODE REVIEW

File: deploy/kubernetes/v1.13/ibm-block-csi-driver.yaml [hunk 1]
```diff
@@ -1,499 +0,0 @@
-# ==============================================================
```
Comment (ArbelNathan) [inline deploy/kubernetes/v1.13/ibm-block-csi-driver.yaml:1]: why delete this and v1.14 file?

Comment (lisyukanna) [reply deploy/kubernetes/v1.13/ibm-block-csi-driver.yaml:1]: it is defined as part of the task

**Review by oriyarde — APPROVED** [review]
LGTM

**Review by ArbelNathan — APPROVED** [review]

File: deploy/kubernetes/examples/demo-storageclass.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: demo-storageclass
+provisioner: block.csi.ibm.com
+parameters:
+  SpaceEfficiency: deduplicated   # Optional.
+  pool: demo-pool
```
Comment (oriyarde) [inline deploy/kubernetes/examples/demo-storageclass.yaml:8]: @ArbelNathan do you think we should mention that this is only mandatory for ds8k or something?

**Pull request merged**